src/org/traccar/protocol/Tk103ProtocolDecoder.java

```diff
@@ -223,6 +415,11 @@ protected Object decode(
             return decodeNetwork(channel, remoteAddress, sentence);
         }
 
+        Position positionWifi = decodeLbsWifi(channel, remoteAddress, sentence);
+        if (positionWifi != null) {
```

Comment:
Can this be identified by message type instead?

Reply:
intention was that some other tk103-like devices will have the same format for wifi-positioning, but if not needed i can check for specific T580W commands. The other problem is that command depends from alarm code, but i can check at least known commands if needed...